tests/functional/spec/question_description.spec.js

```diff
@@ -1,8 +1,35 @@
 import NameBlockPage from "../generated_pages/question_description/name-block.page.js";
+import DescriptionBlockPage from "../generated_pages/optional_guidance_and_description/description-block.page";
+import RadioPage from "../generated_pages/optional_guidance_and_description/mandatory-radio.page";
+import RadioPageTwo from "../generated_pages/optional_guidance_and_description/mandatory-radio-two.page";
+import IntroductionPage from "../generated_pages/question_guidance/introduction.page";
+import GuidancePage from "../generated_pages/question_guidance/block-test-guidance-title.page";
 
 describe("Question description", () => {
   it("Given a question description has been set in the schema as an array, When it is rendered, Then it is displayed correctly as multiple paragraph attributes", async () => {
     await browser.openQuestionnaire("test_question_description.json");
     await expect(await $(NameBlockPage.questionTitle()).getHTML()).to.contain("<p>Answer the question</p><p>Go on</p>");
   });
 });
+
+describe("Optional question description and guidance", () => {
+  it("Given a question description has been set in the schema, When the value to be displayed is None, Then it is not rendered on the page", async () => {
+    await browser.openQuestionnaire("test_optional_guidance_and_description.json");
```

Comment:
Would we want a before() hook when launching the questionnaire?

Reply:
Don't think I need that as there it's all being done within one test, would need before or before each if i had multiple cases for the same schema